.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      # TODO: Switch to tighter version-number matching.
+      - '[0-9]+.*' # Push events to tags starting with numbers
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      # Set up Node
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+          registry-url: 'https://registry.npmjs.org'
+      # Install dependencies
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Build project
+        # I couldn't quickly figure out how to get just the tag name in both a `run` context and as an input to the release asset action, below.
+        # See https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/td-p/31595
+        # We need to move back to /lib after zipping so that npm publish can also find the assets.
+        run: |
+          yarn build
+          mv lib loam
+          zip -r -9 loam.zip loam CHANGELOG README.md LICENSE package.json
+          mv loam lib
```

Comment:
I think I was coming down with a cold (not COVID-19!) at the time I wrote this, so I think I should revisit this, but the motivation was so that someone unzipping the file doesn't end up with a generically-named `lib` folder. I think I can figure out something better, however; I didn't spend a lot of time thinking about this.

Reply:
Upon further research, I don't think doing this is necessary or expected -- most other projects I've checked simply rely upon the GitHub default behavior, which is to include `.zip` and `.tar.gz` files for the source code at the commit of the release. The library is available from `npm`, which should be sufficient. So I'm just going to remove this.